init.go

```diff
@@ -23,17 +22,13 @@ tasks:
     silent: true
 `
 
-const defaultTaskfileName = "Taskfile.yml"
-
 // InitTaskfile Taskfile creates a new Taskfile
-func InitTaskfile(w io.Writer, dir string) error {
-	f := filepathext.SmartJoin(dir, defaultTaskfileName)
-
-	if _, err := os.Stat(f); err == nil {
+func InitTaskfile(w io.Writer, filepath string) error {
```

Comment:
I prefer to avoid `filepath` as a variable name as it conflicts with the stdlib package of the same name. We're not currently importing it here, but it can lead to confusing error down the line. IMO the std package name was an awful choice, but here we are.

`filePath` is fine if you want to keep the name though. Or use the same name (`path`) as you used in the caller.